src/pages/en/core-concepts/astro-components.md

```diff
@@ -348,18 +348,30 @@ They can be used to style your components, and all style rules are automatically
 
 To send JavaScript to the browser without [using a framework component](/en/core-concepts/framework-components) (React, Svelte, Vue, Preact, SolidJS, AlpineJS, Lit) or an [Astro integration](https://astro.build/integrations/) (e.g. astro-XElement), you can use a `<script>` tag in your Astro component template and send JavaScript to the browser that executes in the global scope.
 
+By default, `<script>` tags are hoisted and `type="module"`. They will be placed in the `<head>` of the document and will be bundled with your imports. This is useful for sending client-side scripts to the browser that are not tied to a specific component.
+
+> ⚠️ TypeScript support is not currently available for client-side scripts, but we [are discussing](https://github.com/withastro/rfcs/discussions/193) adding this feature.
```

Comment:
I’m not sure we usually include links to RFCs in the docs? Maybe we could say this instead:

```suggestion
> ⚠️ You can’t currently write TypeScript in client-side scripts, but you _can_ import a Typescript file if you prefer writing with that syntax.
```

Reply:
If I remember correctly Fred used the RFC link on the docs (I can't remember which page) to endorse users to check out by themselves the current status

Reply:
You're both right! 💜  @Chrissdroid @delucis 

Fred has used RFC's as placeholders in the past, but they really are placeholders and are updated once there's something better/official to link them to. So, it's not unheard of, but perhaps best left to things that:

-  *need* but don't yet have any documentation (I think that was the case in the past)
- are realistically coming soon? While well received, this RFC is only a few days old. So even if agreed-upon, I'm not sure how long it would take to make it happen?
- are key/core features that people expect or need, and might be put off by it not existing yet

I'm OK either way, link or no link, as long as we think it's reasonably justified by a criteria like above. Otherwise, the docs could be full of links to things in various stages of development, that may or may not happen, and then require US to manually keep track of all these things and update pages separately from these RFCs being closed.

Reply:
I think we only link to _accepted_ RFCs for more information (like on deprecating the Markdown component). I still feel providing the workaround here (that you can write a `.ts` file and import that) is probably more helpful than the RFC.

Reply:
Agree with @delucis